packages/oui-back-button/src/back-button.html

```diff
@@ -1,10 +1,15 @@
-<button type="button"
+<button ng-if="!$ctrl.href"
+        type="button"
         class="oui-button oui-button_icon-alone"
         ng-attr-aria-label="{{:: $ctrl.ariaLabel }}"
         ng-attr-id="{{:: $ctrl.id }}"
         ng-attr-name="{{:: $ctrl.name }}"
-        ng-click="$ctrl.onBtnClick()"
-        type="button">
+        ng-click="$ctrl.onBtnClick()">
   <i class="oui-icon oui-icon-chevron-left" aria-hidden="true"></i>
 </button>
+<a ng-if="$ctrl.href"
```

Comment:
We should go with `ng-if="::!!$ctrl.href"` if it's one-time binded

Reply:
Yes u'right!